lti_consumer/lti_1p3/consumer.py

```diff
@@ -625,6 +721,173 @@ def check_and_decode_deep_linking_token(self, token):
+
+        # TODO: We MUST perform other forms of validation here.
+        # TODO: We MUST validate the verified_user claim if it is provided, although it is optional to provide it.
+        #       An Assessment Platform MAY reject the Start Assessment message if a required identity claim is missing
+        #       (indicating that it has not been verified by the Proctoring Tool).  Which identity claims a
+        #       Proctoring Tool will verify is subject to agreement outside of the scope of this specification but, at a
+        #       minimum, it is recommended that Proctoring Tools performing identity verification are able to verify the
+        #       given_name, family_name and name claims.
+        #       See 3.3 Transferring the Candidate Back to the Assessment Platform.
+
+        # Check Required LTI Claims
+        # -------------------------
+
+        # Check that the response message_type claim is "LtiStartAssessment".
+        # message_type = proctoring_response.get("https://purl.imsglobal.org/spec/lti/claim/message_type")
+        # if message_type is None:
+        #     raise exceptions.MissingRequiredClaim("Token is missing required message_type claim.")
+        # if message_type != "LtiStartAssessment":
+        #     raise exceptions.InvalidClaimValue("Token's message_type claim should be LtiStartAssessment.")
+
+        claim_key = "https://purl.imsglobal.org/spec/lti/claim/message_type"
+        self._check_token_claim(
+            proctoring_response,
+            claim_key,
+            "LtiStartAssessment",
+            f"Token's {claim_key} claim should be LtiStartAssessment."
+        )
+
+        # Check that the response version claim is "1.3.0".
+        # version = proctoring_response.get("https://purl.imsglobal.org/spec/lti/claim/version")
+        # if version is None:
+        #     raise exceptions.MissingRequiredClaim("Token is missing required version claim.")
+        # if version != "1.3.0":
+        #     raise exceptions.InvalidClaimValue("Token's version claim should be 1.3.0.")
+
+        claim_key = "https://purl.imsglobal.org/spec/lti/claim/version"
+        self._check_token_claim(
+            proctoring_response,
+            claim_key,
+            "1.3.0",
+            f"Token's {claim_key} claim should be 1.3.0."
+        )
+
+        # Check that the response deployment_id claim is "1". We do not support multi-tenancy,
+        # so this should always be "1".
+        # deployment_id = proctoring_response.get("https://purl.imsglobal.org/spec/lti/claim/deployment_id")
+        # if deployment_id is None:
+        #     raise exceptions.MissingRequiredClaim("Token is missing required deployment_id claim.")
+        # if deployment_id != "1":
+        #     raise exceptions.InvalidClaimValue("Token's deployment_id claim is not correct.")
+
+        claim_key = "https://purl.imsglobal.org/spec/lti/claim/deployment_id"
```

Comment:
I wouldn't hardcode this check here and now - the worst it can do with the current implementation is just fail a validation and stop the tool from working.